include/swift/AST/DiagnosticsSema.def

```
@@ -9060,5 +9060,7 @@ ERROR(unsafe_self_dependent_result_attr_on_invalid_decl,none,
 
 REMARK(macro_expansion_line, none, "macro content: |%0|", (StringRef))
 
+WARNING(empty_catch_block, none, "catch block is empty, errors will silently be ignored", ())
```

nit: a comma feels slightly wrong here... maybe a semicolon would be more fitting? @xwu any thoughts on this?

---

I have changed it to "empty catch block silently ignores errors; consider using 'try?' or 'catch _' to explicitly ignore".

---

Nits: (1) ideally, I think, you’d add this line to the file alongside the other diagnostics about do…catch and not just at the end of the file; (2) re the wording, I don’t think the block is doing the ignoring; perhaps it is right to say that it silences all errors, but that is also at best just restating what the code does in English, so I’m also not sure that it’s much helpful—writing is tough!; (3) don’t we have a policy now that all new diagnostics should go into a named diagnostic group?